src/mono/wasm/runtime/dotnet.d.ts

```diff
@@ -332,6 +340,10 @@ interface ResourceGroups {
     readonly libraryInitializers?: ResourceList;
     readonly extensions?: BootJsonDataExtension;
     readonly runtimeAssets: ExtendedResourceList;
+    readonly vfs?: {
+        [virtualPath: string]: ResourceList;
+    };
+    readonly remoteSources?: string[];
```

Comment:
I have changed the usage to

```js
moduleConfig.remoteSources = (resourceLoader.bootConfig.resources as any).remoteSources;
```

Is there any better way?

Reply:
I think that we never populate it in the WasmAppBuilder and probably nobody ever does. So, maybe you don't even have to integrate it here. Otherwise, this is good enough.

Reply:
It's defined as `@(WasmRemoteSources)`, but no project on github use it